docs/src/content/faq.mdx

```diff
@@ -48,27 +81,275 @@ After receiving the NFT, follow these steps to set up the Icebreaker platform:
+- `3000/tcp` (or custom port): For incoming API requests from Blockfrost
+- `8081/tcp`: Prometheus metrics endpoint for monitoring (optional but recommended)
 
-If you've received your NFT but don't have channel access yet, post a message on [Discord](https://discord.com/invite/inputoutput) in the `#ask-blockfrost` channel for assistance.
+Ensure these ports allow incoming connections from the Blockfrost server IPs documented [here](/usage/network#firewall).
 
-## Do I need confirmation after receiving the NFT to proceed?
+You can verify port accessibility using:
+```bash
+sudo ufw allow 3000/tcp
+sudo ufw allow 8081/tcp
+```
 
-No, you don't need additional confirmation after receiving the NFT.
-Once the NFT is in your wallet, you can proceed with installation and setup.
-However, you'll need access to the private Icebreakers channel for complete documentation, which is granted after notifying the team that you've received your NFT.
+### How do I build `blockfrost-platform` from source?
 
-## Can I transfer my Icebreaker NFT to another wallet address?
+Refer to the [installation instructions](/installation/source) for detailed steps on building from source.
+This is often recommended over using pre-built installers if you encounter issues with the provided binaries.
 
-Yes, you can transfer your Icebreaker NFT to another wallet address by following these steps:
+### Do I need a static public IP address for my Icebreaker node?
+
+Yes, a static public IP is strongly recommended for reliably running an Icebreaker node.
+
+The Blockfrost platform currently identifies and monitors nodes based on their consistent public IP. Without a static IP, you risk connection issues and flapping.
+
+A future update will allow dynamic IPs by having the node proactively report its address. See [this GitHub issue](https://github.com/blockfrost/blockfrost-platform/issues/178) for details on that roadmap.
+
+In the interim, using a static IP is best. If not possible, a dynamic DNS service or VPN may help provide a stable hostname to IP mapping.
+
+### What should I do if my public IP address changes frequently?
+
+Frequent public IP address changes can disrupt the connection between your Icebreaker and the Blockfrost servers.
+A future update will address this by allowing connections from behind NATs (see [this GitHub Issue](https://github.com/blockfrost/blockfrost-platform/issues/178)).
+In the meantime, consider using a static IP address or a dynamic DNS service.
+
+### How do I run blockfrost-platform with only IPv4?
+
+While a dedicated `-4` CLI switch isn't currently available, you can work around IPv6 issues by adding an entry to your `/etc/hosts` file that maps the Blockfrost server address to its IPv4 address.
+
+### Should I run my Icebreaker node on bare metal or in a VM?
+
+Either bare metal or VM-based Icebreaker nodes will work fine.
+The choice mainly depends on your available infrastructure and operational preferences.
+
+Benefits of bare metal:
```

Comment:
hmmmm, these pros and cons are arguable. I would skip this info since it's not our domain.
And blockfrost-platform will run on the same machine that runs the cardano-node anyways.

Reply:
Alright, thanks for your feedback about that. I'll remove this entry.